research/build_crypto_strategy_notebook.py

```
@@ -0,0 +1,165 @@
+- Account-level fee tier, taxes, delisting, and exchange/API outage behavior are not included.
+"""
+        ),
+        nbf.v4.new_markdown_cell("## Takeaways"),
+        nbf.v4.new_code_cell(
+            """as_of = study['snapshot']['as_of_utc'].min()
+conservative = study['risk_scenarios'].query('target_vol_pct <= 25').sort_values('selection_sharpe_2021_2023', ascending=False).iloc[0]
+futures = study['aggregate'][study['aggregate']['strategy'].str.contains('long_short')]
+rsi = study['aggregate'].query("strategy == 'rsi_pullback_long_cash'").iloc[0]
+display(Markdown(f'''Results below are observations from the committed study run through **{as_of}** and will change after a data refresh.
+
+1. **Paper-trading candidate:** long/cash 90-day momentum + 200-day trend filter, BTC/ETH/SOL 60/30/10, no leverage. Among the conservative 20% and 25% rows, the 2021–2023 training-only Sharpe selects **{conservative.target_vol_pct:.0f}%** (training Sharpe **{conservative.selection_sharpe_2021_2023:.2f}**). Its untouched 2024-current holdout return is **{conservative.return_2024_current_pct:.1f}%**; full-sample CAGR **{conservative.cagr_pct:.1f}%** and max drawdown **{conservative.max_drawdown_pct:.1f}%** are descriptive, not selection inputs.
```

Ruff (0.16.2)

[warning] String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)